src/components/builder/values/items-expression.tsx

```diff
@@ -1,12 +1,96 @@
-import {JSONObject} from '@open-formulieren/types/lib/types';
+import {AnyComponentSchema} from '@open-formulieren/types';
+import {JSONObject, JSONValue} from '@open-formulieren/types/lib/types';
 import {useFormikContext} from 'formik';
+import {useContext} from 'react';
 import {FormattedMessage} from 'react-intl';
 
 import JSONEdit from '@/components/JSONEdit';
 import {Component, Description} from '@/components/formio';
+import {BuilderContext, VariableDefinition} from '@/context';
 
 const NAME = 'openForms.itemsExpression';
 
+const dataTypeForComponent = (component: AnyComponentSchema): JSONValue => {
```

Comment:
would it make sense to add this to the `infernologic` library itself? In a `contrib/formio` package or something like that? That would also make it more easily shared between the builder & backend.